shared/js/ui/base/localize.es6.js

```diff
@@ -1,37 +1,30 @@
 import i18next from 'i18next'
 import ICU from 'i18next-icu'
-import config from '../../../data/constants.js'
 
-const supportedLocales = config.supportedLocales
+const localeResources = require('../../../locales/*/*.json', { mode: 'list' })
```

Comment:
Since we aren't relying on a static list of locales here, we shouldn't merge this until we're ready to release the localized extensions (or revert to using the list from `constants.js` for now)